### ISSUE TITLE
Registry authentication not working

### ISSUE THREAD

**Issue by gardar** [Polpetta/packer-plugin-podman#45]
I'm getting errors when trying to pull images from a registry that requires authentication, looking at the source it seems that the login feature from the docker plugin has not been fully ported over.

**Comment by Polpetta** [comment]
Yes, you're indeed correct, I just ported the bare minimum that time! I'll check your pull request and add it if everything's ok 👍
